### ISSUE TITLE
[Bug]: Manager app does not work on "non-touch devices" like tv-boxes

### ISSUE THREAD

**Issue by AndKe** [YTVanced/VancedManager#705]
### Device

Device: Xiaomi TV Stick
Operating System: Android
Android Version: 9

### Version

current, fresh downloaded version

### Bug Description

the "manager" , from the pointless "lets get started" (not)  is not compatible with arrow keys/ok/back and other navigation buttons, and can't be used.
Irony: beside the bad manager app, every related app works fine without touch interface.

### Additional checks

- [X] This is a bug in Vanced Manager and NOT YouTube Vanced/YouTube Vanced Music/Vanced microG.

**Comment by kjlawrencelaw** [comment]
So use with the wired mouse or touchpad (through Bluetooth)

**Comment by AndKe** [comment]
yes, that is a workaround for those with such BT device, it is also possible to install and use remote-control software, but both are more than users expect.
It's better to make a normal UI, and produce a better user experience.

**Comment by X1nto** [comment]
Please wait for Manager 3.0 that should come out soon, it will have all things fixed
